packages/@expo/cli/bin/cli.ts

```diff
@@ -27,6 +27,7 @@ const commands: { [command: string]: () => Promise<Command> } = {
   export: () => import('../src/export/index.js').then((i) => i.expoExport),
   'export:web': () => import('../src/export/web/index.js').then((i) => i.expoExportWeb),
   'export:embed': () => import('../src/export/embed/index.js').then((i) => i.expoExportEmbed),
+  'verify-native-assets': () => import('../src/verifyNativeAssets/index.js').then((i) => i.expoVerifyNativeAssets),
```

Comment:
Right now, `verify-native-assets` will appear in `npx expo --help` output. Since this looks like an internal command, you'll also need to explicitly [pull this command out here](https://github.com/expo/expo/blob/c8b3f7bcf9f50980478e41e831f7b70550d8a2b4/packages/%40expo/cli/bin/cli.ts#L90-L97) to hide it.

This command name also seems to deviate a bit from the format we use in other commands. E.g. `npx expo asset:validate` etc.

Reply:
Command name is now `assets:verify`